book/src/kubernetes-changelog.md

```diff
@@ -3,6 +3,28 @@
 This page summarizes major CSI changes made in each Kubernetes release. For
 details on individual features, visit the [Features section](features.md).
 
+## Kubernetes 1.23
+* Beta
+    * Add Fsgroup Mount
+* Alpha
+    * Recover from Expansion Failure
+    * Auto delete PVCs created by StatefulSet
+    * Honor PV Reclaim Policy
+    * RBD CSI Migration
+    * Portworx CSI migration
+
+## Kubernetes 1.22
```

Comment:
Can you add the `### Features` subheading under each release too?

Reply:
added it